dynatrace_extension/cli/create/extension_template/extension_name/__main__.py.template

```diff
@@ -5,6 +5,9 @@ class ExtensionImpl(Extension):
 
     def initialize(self):
         self.extension_name = "%extension_name%"
+        self.extension_version = self.activation_config.version
+        self.logger.name = self.extension_name
+        self.logger.info(f"Initializing {self.extension_name} {self.extension_version}")
```

Comment:
The extension version will be logged twice, it is already using the extension version, and there is already a way to get the version, I don't think we should introduce one more.

https://github.com/dynatrace-extensions/dt-extensions-python-sdk/blob/5535ea11a319cd9e1821c23c2b2a12f7c40c2bf4/dynatrace_extension/sdk/extension.py#L242

Can we maybe add this to the `__init__` method and hide it from the user?

Reply:
`self.get_version()` doesn't seem to print extension version but rather sdk version which was my problem as it doesn't seem as relevant.

Reply:
I like the __init__ proposal. I'll apply that change